pkg/v1/config/doc.go

```diff
@@ -1,5 +0,0 @@
-// Copyright 2021 VMware, Inc. All Rights Reserved.
-// SPDX-License-Identifier: Apache-2.0
-
-// Package config provides functions for reading and writing client configuration.
-package config
```

Comment:
What are these empty `doc.go` files for?

Reply:
I think based on my understanding doc.go file within a package is used to write go documentation for specific to the package.